package.json

```diff
@@ -10,21 +10,23 @@
   },
   "homepage": "https://solarwallet.io/",
   "repository": "github:satoshipay/solar",
-  "main": "./electron/lib/app.js",
+  "main": "./electron/lib/index.prod.js",
```

Comment:
I changed this to `index.prod.js` because I guess this is the right place to declare our production index file? I was not 100% sure about this, but I did not find any usages of our old `electron/index.js` so I thought this was by accident.